tests.py

```diff
@@ -177,15 +188,16 @@ def testShowContainers(self):
 
   def testGetLayerInfo(self):
     """Tests the Storage.GetLayerInfo function on a AUFS storage."""
-    layer_info = self.storage.GetLayerInfo(
-        'sha256:{0:s}'.format(self.image_id))
+    layer_info = self.de_object.storage_object.GetLayerInfo(
+        'sha256:'
+        '7968321274dc6b6171697c33df7815310468e694ac5be0ec03ff053bb135e768')
     self.assertEqual('2017-01-13T22:13:54.401355854Z', layer_info['created'])
     self.assertEqual(['/bin/sh', '-c', '#(nop) ', 'CMD ["sh"]'],
                      layer_info['container_config']['Cmd'])
 
   def testShowRepositories(self):
     """Tests the Storage.ShowRepositories function on a AUFS storage."""
```

Comment:
"Test that ShowRepositories returns a correctly formatted info string"

Reply:
See #24
I'll make test docstrings more explicit at the same time I split unit tests.